pykafka/balancedconsumer.py

```diff
@@ -301,12 +302,14 @@ def _decide_partitions(self, participants):
         :type participants: Iterable of str
         """
         # Freeze and sort partitions so we always have the same results
-        p_to_str = lambda p: '-'.join([p.topic.name, str(p.leader.id), str(p.id)])
+        p_to_str = lambda p: b'-'.join([
+            get_bytes(p.topic.name), bytes(p.leader.id), bytes(p.id)]
```

Comment:
This might be more readable if the lambda just returns a plain tuple, without any bytes or str coercions.